paddle/fluid/inference/api/mkldnn_quantizer.cc

```diff
@@ -68,10 +68,13 @@ bool AnalysisPredictor::MkldnnQuantizer::CalculateScales() {
             if (is_output) {
               if (op->Type() == "conv2d") {
                 // output of conv2d with relu must be unsigned
-                is_unsigned = (op->HasAttr("fuse_relu") &&
-                               boost::get<bool>(op->GetAttr("fuse_relu"))) ||
-                              (op->HasAttr("fuse_brelu") &&
-                               boost::get<bool>(op->GetAttr("fuse_brelu")));
+                std::string fuse_activation =
+                    op->HasAttr("fuse_activation")
+                        ? boost::get<std::string>(
+                              op->GetAttr("fuse_activation"))
+                        : "";
+                is_unsigned =
```

Comment:
Please consider introducing method in Op or OpDesc

```C++
template <typename T>
T GetAttrIfExists(std::string AttrName)
{
   T result;
    if(HasAttr(AttrName) {
        result = GetAttr(AttrName)
   }
   return result;
}
```

since the pattern of
```
std::string fuse_activation =
                    op->HasAttr("fuse_activation")
                        ? boost::get<std::string>(
                              op->GetAttr("fuse_activation"))
                        : "";
```
which is often recurring, could be simplified to `std::string fuse_activation = op_->GetAttrIfExists("fuse_activation")`

Reply:
Great idea! This function will take use in many places.